src/TMxpMusicTagHandler.cpp

```diff
@@ -0,0 +1,90 @@
+/***************************************************************************
+ *   Copyright (C) 2020 by Mike Conley - sousesider[at]gmail.com           *
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ *   This program is distributed in the hope that it will be useful,       *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU General Public License for more details.                          *
+ *                                                                         *
+ *   You should have received a copy of the GNU General Public License     *
+ *   along with this program; if not, write to the                         *
+ *   Free Software Foundation, Inc.,                                       *
+ *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
+ ***************************************************************************/
+
+#include "TMedia.h"
+#include "TMxpMusicTagHandler.h"
+#include "TMxpClient.h"
+
+TMxpTagHandlerResult TMxpMusicTagHandler::handleStartTag(TMxpContext& ctx, TMxpClient& client, MxpStartTag* tag)
+{
+    TMediaData mediaData {};
+
+    mediaData.setMediaFileName(tag->getAttributeValue("FName"));
+    mediaData.setMediaType(TMediaData::MediaTypeMusic);
+
+    QString volume = tag->getAttributeValue("V");
```

Comment:
Use hasAttribute("V") for optional attributes. Trying to access a non-existing attribute may lead to a crash.

Reply:
Done, thanks!